web-client/package.json

```diff
@@ -59,7 +59,8 @@
     "start": "cross-env SKIP_PREFLIGHT_CHECK=true HTTPS=true react-app-rewired start",
     "clean": "rm -rf build",
     "build": "cross-env react-app-rewired build",
-    "test": "react-app-rewired test --watchAll --env=jest-environment-jsdom-sixteen",
+    "test": "react-app-rewired test --env=jest-environment-jsdom-sixteen",
+    "test:ci": "cross-env CI=true react-scripts test --env=jest-environment-jsdom-sixteen",
```

Comment:
Just curious, does it it work if you do not call `react-scripts` here but `react-app-rewired`?
For example: ` "test:ci": "cross-env CI=true react-app-rewired test --env=jest-environment-jsdom-sixteen"`
IF it's working as is, don't change it. I just think the override issues is caused by using vanilla react-scripts here.

Reply:
oh, yea. I might have overlooked this. let me try this out

Reply:
yup, it seems to do the trick. I will push it now. thanks 👍